packages/spark-core/base/_links.scss

```diff
@@ -50,20 +50,21 @@
   &:focus,
   &:visited {
     border-bottom: $sprk-link-has-icon-hover-border-bottom;
+    color: $sprk-link-has-icon-color;
```

Comment:
this var should follow the other hover state vars and have the word "hover" in the var name.

Reply:
This variable is the default color/stroke for the icon as well as the hover/visited/active color for the text. Should this be two different variables?

Reply:
@bobwise yes, we'll want to make sure those two things are able to be changed since we can change all the other links in that way.